src/configuration/app-containers.md

```diff
@@ -254,35 +249,35 @@ web:
             ...
 ```
 
-It has a few subkeys, which are:
+There are subkeys listed below, some of which are used by web.locations as follows:
```

Comment:
This sentence feels far clumsier to me.  Honestly I'd leave this line as is.  I can't think of a less confusing way to word it.